styles/Editor.css

```diff
@@ -939,6 +939,44 @@
   font-size: 75%;
 }
 
+.ProseMirror .table-footer {
+  font-size: 14px;
+}
+.ProseMirror .table-footer p {
+  margin-bottom: 0px;
+  margin-top: 0px;
+}
+
+.ProseMirror .table-footer  {
+  font-size: 14px;
+  padding-top: 8px;
+}
+
+.ProseMirror .table-footer p {
+  margin-bottom: 0px;
+  margin-top: 0px;
+}
+
+.ProseMirror .table-footer .footnote-selected, .table-footer > div:has(> p.footnote-selected) > p  {
+  background-color: #f2fbfc;
+  padding: 4px;
+}
+.ProseMirror .table-footer .footnote-element .footnote-selected {
+  border: 1px solid #bce7f6;
+}
+.table-footer > div > p {
```

Comment:
No need. you already have `padding-top: 8px;` for the `.ProseMirror .table-footer`

Reply:
this is needed for general notes